### PR TITLE
flamin' hot cheesie honkers v2: the rehonkening

### PR DESCRIPTION
**Pull request by seekssss**
## About The Pull Request

second time's the charm

check that nothing is broken. i had to rewrite this entire code while trying to fix the merge conflict.

there are some odd line removals here that I am not sure why they're happening but im gonna assume its nothing <:^)

## Why It's Good For The Game

hot cheeto

## Changelog
:cl:
add: added hot cheesies
add: added new cheesie trash sprites
/:cl:

### CODE REVIEW

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -143,7 +143,7 @@
 	product_records = null
 	return ..()
 
-/obj/machinery/vending/legacy_ex_act(severity)
+/obj/machinery/vending/ex_act(severity)
```
Comment (TheLordME) [inline code/game/machinery/vending.dm:146]: unless you changed this under silicons instructions please revert to legacy_ex_act

File: code/modules/food/food/snacks.dm [hunk 2]
```diff
@@ -358,6 +358,20 @@
 	reagents.add_reagent("sugar", 4)
 	bitesize = 6
 
+/obj/item/reagent_containers/food/snacks/candy/flaminhotcheesiehonker
+	name = "flamin' hot cheesie honkers"
+	desc = "Found in an abandoned warehouse, probably not fit for safe consumption, still delicious. Now with 50% more mouth-melting dust!"
+	icon_state = "hotcheeziehonker"
+	trash = /obj/item/trash/candy/flaminhotcheesiehonker
+	nutriment_amt = 5
+	nutriment_desc = list("capsaicin" = 2, "cheese" = 3, "chips" = 2)
```
Comment (TheLordME) [inline code/modules/food/food/snacks.dm:367]: is "chips" a reagent we have?

File: code/game/machinery/vending.dm [hunk 3]
```diff
@@ -661,7 +661,6 @@
 					/obj/item/reagent_containers/food/drinks/bottle/champagne/jericho = 1,
 					/obj/item/reagent_containers/food/drinks/bottle/champagne = 3,
 					/obj/item/reagent_containers/food/drinks/bottle/coconutmilk = 5,
-					/obj/item/reagent_containers/food/drinks/bottle/unathijuice = 2,
```
Comment (TheLordME) [inline code/game/machinery/vending.dm:664]: Why are we removing this? its only been a recent addition

File: code/game/machinery/vending.dm [hunk 4]
```diff
@@ -711,13 +710,13 @@
 					/obj/item/reagent_containers/food/snacks/sosjerky = 6,/obj/item/reagent_containers/food/snacks/no_raisin = 6,/obj/item/reagent_containers/food/snacks/spacetwinkie = 6,
 					/obj/item/reagent_containers/food/snacks/cheesiehonkers = 6, /obj/item/reagent_containers/food/snacks/tastybread = 6, /obj/item/reagent_containers/food/snacks/skrellsnacks = 3,
 					/obj/item/reagent_containers/food/snacks/baschbeans = 6, /obj/item/reagent_containers/food/snacks/creamcorn = 6, /obj/item/reagent_containers/hard_candy/lollipop = 6,
-					/obj/item/reagent_containers/food/snacks/spunow = 6, /obj/item/reagent_containers/food/snacks/glad2nut = 6, /obj/item/reagent_containers/food/snacks/natkat = 6)
-	contraband = list(/obj/item/reagent_containers/food/snacks/syndicake = 6,/obj/item/reagent_containers/food/snacks/unajerky = 6,)
+					/obj/item/reagent_containers/food/snacks/spunow = 6, /obj/item/reagent_containers/food/snacks/glad2nut = 6, /obj/item/reagent_containers/food/snacks/candy/flaminhotcheesiehonker = 6)
```
Comment (TheLordME) [inline code/game/machinery/vending.dm:713]: again you overwrote a seemingly okay entry, same thing happend on like 720

**Review by TheLordME — CHANGES_REQUESTED** [review]

File: code/game/machinery/vending.dm (hunk 3 above)
Comment (out-of-phaze) [reply code/game/machinery/vending.dm:664]: It's a merge artifact issue. They resolved the merge improperly.

**Comment by seekssss** [comment]
i will get back to this when i have time to wrack my head against coding. sorry for wasting time, I'm still figuring some stuff out.

**Pull request closed without merging**